packages/react-core/src/components/TextArea/examples/TextAreaAutoResizing.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { TextArea } from '@patternfly/react-core';
+
+export const TextAreaAutoResizing: React.FunctionComponent = () => (
+  <TextArea aria-label="auto resizing text area example" autoResize />
+);
```

Comment:
Similar to the above comments, for consistency it might be good to add state for the text area value similar to the other non-uncontrolled examples. @wise-king-sullyman do you have any preference about that?

Reply:
I wouldn't be against it, not sure if the examples were originally done in this manner for some reason though.